<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Grey_f1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Grey Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Black_White_f2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Black &amp; White Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Invert_f3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Merge_f4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge 2 Photos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Flip_f5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rotate_f6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Brightness_f7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness Editor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Crop_f8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Frame_f9">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Edges_f10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Resize_f11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Blur_f12">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sun_f13">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SunLight Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OilPainting_f14">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OilPainting Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TV_f15">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TV Noise Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Purple_f16">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Purple Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InfraRed_f17">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Infra-Red Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Skew_f18">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Skew Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WhiteRed_e1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>White Red Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Blood_e2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Blood Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RedGreen_e3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Green Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="XRay_e4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X-Ray Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetEffects">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Effects</string>
    </property>
   </widget>
   <widget class="QLabel" name="Picture">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>35</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MergeC1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MergeC2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choice 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlipV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlipH">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QSlider" name="BrightSl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RotateL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RotateR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrameB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrameR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrameBW">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue &amp; White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrameG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Green Frame</string>
    </property>
   </widget>
   <widget class="QSlider" name="BlurSl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SkewSl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>80</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="OilSlow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Oil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OilFast">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced Oil</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
